drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/main/java/org/drools/workbench/screens/scenariosimulation/client/utils/ConstantHolder.java

```diff
@@ -31,4 +31,6 @@
     // GWT doesn't support Java 8 LocalDate
     public static final String LOCALDATE_SIMPLE_NAME = "LocalDate";
     public static final String LOCALDATE_CANONICAL_NAME = "java.time.LocalDate";
+
+    public static final String DMN_DATE = "date";
```

Comment:
@gitgabrio @manstis me and @dupliaka are not happy I reintroduced an existing constant **date**. It is already for example in the [BuiltInType](https://github.com/kiegroup/kie-wb-common/blob/master/kie-wb-common-dmn/kie-wb-common-dmn-api/src/main/java/org/kie/workbench/common/dmn/api/property/dmn/types/BuiltInType.java). Would be possible from your point of view to add a dependency to **kie-wb-common-dmn-api** module? If that is not preferable, is it possible to move **BuiltInType** to some shared module, for example **drools-workbench-models**? Or is it really best to introduce the date string constant here?

Reply:
@jomarko 
Hi. One thing that I would like to avoid as the pest - as general rule - is to add other dependencies. 
Beside that, I surely won't do that for a single string (as this seems the case).
Having said that, if there is not already a constant "date" to reuse in all our current classpath, I would not see any issue in using this one. 
The only issue I see is that - in general - we should name objects based on what they _are_, and not on what they _are used for_. In that line, `DMN_DATE` is just _date_ - but "DMN" is only the way you currently use it.
So, if there is not already a  _date_ String somewhere, please keep using this (maybe rename it as "DATE" or something more generally re-usable).
About moving **BuiltInType**, it brings with it _org.kie.workbench.common.dmn.api.property.dmn.QName_, and who knows what in cascade - and all of that only for a single string: again, I do not see a valid reason to do that.

( Out of curiosity - please look that https://github.com/kiegroup/kie-wb-common/blob/master/kie-wb-common-dmn/kie-wb-common-dmn-api/pom.xml: my point of view about dependencies  should be more clear)